src/providers/tenderly-simulation-provider.ts

```diff
@@ -1,3 +1,6 @@
+import http from 'http';
+import https from 'https';
```

Comment:
That breaks react-native:

```
Bundling failed 37152ms
The package at "node_modules/@uniswap/smart-order-router/build/main/providers/tenderly-simulation-provider.js" attempted to import the Node standard library module "http".
It failed because the native React runtime does not include the Node standard library.
```